ament_cppcheck/bin/ament_cppcheck

```diff
@@ -41,7 +42,11 @@ def main(argv=sys.argv[1:]):
         print('No files found', file=sys.stderr)
         return 1
 
-    cppcheck_bin = find_executable('cppcheck')
+    additional_paths = None
+    if os.name == 'nt':
+        # search in location where cppcheck is installed via chocolatey
+        additional_paths = ['C:/Program Files (x86)/Cppcheck']
```

Comment:
Doesn't look like it:

![screen shot 2015-04-06 at 3 18 22 pm](https://cloud.githubusercontent.com/assets/100427/7013672/3709e092-dc70-11e4-8fb3-ee06f96df1dd.png)